x-pack/plugins/security_solution/public/common/components/hover_actions/use_hover_actions.tsx

```diff
@@ -114,7 +116,7 @@ export const useHoverActions = ({
         ownFocus={hoverActionsOwnFocus}
         showOwnFocus={false}
         showTopN={showTopN}
-        timelineId={timelineId ?? timelineIdFind}
+        scopeId={!scopeId ? timelineIdFind ?? tableIdFind : scopeId}
```

Comment:
should we wrap this in `useMemo`? Can't remember how costly `timelineIdFind` and `tableIdFind` are.